apps/src/blockly/addons/cdoFieldPicker.js

```diff
@@ -0,0 +1,129 @@
+import GoogleBlockly from 'blockly/core';
+
+const BUTTON_CORNER_RADIUS = 3;
+const BUTTON_INNER_HEIGHT = 16;
+import {DEFAULT_SOUND} from '@cdo/apps/blockly/constants';
```

Comment:
It feels strange that we have these button style and sound value constants grouped here in this file when they're unrelated to each other and mutually exclusive. If this class is going to be generic, I would think these constants would be passed into the constructor directly from wherever we're making the field. If any of these values were always used, I would feel like it made sense to define or import them here.